src/store/modules/game.js

```diff
@@ -414,5 +414,69 @@ export default {
 				});
 			});
 		},
-	}
-}
\ No newline at end of file
+		async requestPlayOneOffSeven(context, { cardId, index }) {
+			return new Promise((resolve, reject) => {
+				io.socket.get('/game/seven/untargetedOneOff', {
+					cardId,
+					index, // 0 if topCard, 1 if secondCard
+					opId: context.getters.opponent.id,
+				}, function handleResponse(res, jwres) {
+					if (jwres.statusCode !== 200) {
+						return reject(jwres.body.message);
+					}
+					context.commit('setWaitingForOpponentToCounter', true);
+					return resolve();
+				});
+			});
+		},
+		async requestPlayTargetedOneOffSeven(context, { cardId, index, targetId, targetType }) {
+			return new Promise((resolve, reject) => {
+				io.socket.get('/game/seven/targetedOneOff', {
+					cardId,
+					targetId,
+					targetType,
+					index, // 0 if topCard, 1 if secondCard
+					opId: context.getters.opponent.id,
+				}, function handleResponse(res, jwres) {
+					if (jwres.statusCode !== 200) {
+						return reject(jwres.body.message);
+					}
+					context.commit('setWaitingForOpponentToCounter', true);
+					return resolve();
+				});
+			});
+		},
+		async requestPlayOneOffSeven(context, { cardId, index }) {
```

Comment:
Looks like the same function appears twice for `requestPlayOneOffSeven` and `requestPlayTargetedOneOffSeven`